crates/node/src/shell/mod.rs

```diff
@@ -1069,7 +1069,10 @@ where
             TxType::Wrapper(wrapper) => {
                 // Validate wrapper first
                 // Tx gas limit
-                let gas_limit = match Gas::try_from(wrapper.gas_limit) {
+                let gas_scale = get_gas_scale(&self.state)
+                    .expect("Failed to get gas scale from parameters");
```

Comment:
Same here in mempool validation

Reply:
done in 037f62f73b6c7fd1e9a73b8d485a67caa8ec0767